.github/workflows/build-checks.yml

```diff
@@ -5,12 +5,18 @@ name: build-checks
 on:
   push:
   pull_request:
+    types:
+      - opened
+      - synchronize
+      - labeled
+      - unlabeled
 
 permissions:
   contents: read #  to fetch code (actions/checkout)
 
 jobs:
   build-checks:
+    if: ${{ !contains(github.event.pull_request.labels.*.name, 'skip-ci') && (contains(github.event.pull_request.labels.*.name, 'core') || contains(github.event.pull_request.labels.*.name, 'platform/linux') || github.event_name == 'push') }}
```

Comment:
If one has disabled Linux CI because testing something with windows. Is it a good idea to skip this? We need probably an extra label to skip build-checks?

Reply:
The `platform/windows` or `platform/macos` only applies for platform specific changes (e.g buildenv, packing) so these tests would be irrelevant to run since they only applied to the source code. If we decide to add [E2E/UI testing](https://github.com/mixxxdj/mixxx/pull/14360), we might want to reconsider this decision but since we are not ready to do so, it's probably best to be conservative. In case there was a time one wanted to run the `build-checks` anyway, they could always "abuse" the `core` label as an immediate workaround.